exporters/ostream/CMakeLists.txt

```diff
@@ -27,7 +27,7 @@ if(OPENTELEMETRY_INSTALL)
     DIRECTORY include/opentelemetry/exporters/ostream
     DESTINATION include/opentelemetry/exporters
     PATTERN "*.h"
-    PATTERN "log_Exporter.h" EXCLUDE)
+    PATTERN "log_record_exporter.h" EXCLUDE)
```

Comment:
I can't recollect why this exclude was added, do we really need this?

Reply:
Ok. probably copying this separately if WITH_LOGS_PREVIEW is enabled. In that case, should we also exclude `log_record_exporter_factory.h` here?